### DISCUSSION TITLE
vercel部署成功，就是想自定义背景找不到位置，能否在site_config.js那里添加路径？

### DISCUSSION THREAD

**Discussion by chenryace** [spencerwooo/onedrive-vercel-index#703] [Q&A] [answered]
# 感谢作者！！
vercel部署成功，就是想修改背景找不到位置，能否在site_config.js那里添加路径？
**感觉白色虽然耐看就是看久了很腻，特别夜晚很累**，我用原版的Onemanager部署过，可以自定义，在这项目中就是找不到

**Comment by myl7** [comment, marked as answer]
如果是希望修改为暗色模式的话，可以参考 #245 和 #655 的介绍

如果是希望自定义背景图像的话，可能需要自行添加 CSS 进行设置，具体方案可以参考 CSS 文档和 Next 文档
